spec/core/validators/validators.spec.tsx

```
@@ -1,49 +1,82 @@
+    expect(await Validators.MinLength(3)('aaa')).toEqual(null);
+    expect(await Validators.MinLength(3)(null)).toEqual({ minlength: true });
     expect(await Validators.MinLength(3)(undefined)).toEqual({ minlength: true });
+    expect(await Validators.MinLength(3)(emptyString)).toEqual({ minlength: true });
+    expect(await Validators.MinLength(3)(objectWithValue)).toEqual(null);
+    expect(await Validators.MinLength(3)(emptyObject)).toEqual(null);
+    expect(await Validators.MinLength(3)(emptyArray)).toEqual( {minlength: true});
+    expect(await Validators.MinLength(3)(arrayWithValues)).toEqual( {minlength: true});
+    expect(await Validators.MinLength(1)(arrayWithValues)).toEqual(null);
   });
-  it('should validate maxLength correctly', async () => {
-    expect(await Validators.MaxLength(3)('a')).toEqual(null);
-    expect(await Validators.MaxLength(3)('')).toEqual(null);
-    expect(await Validators.MaxLength(3)([])).toEqual(null);
-    expect(await Validators.MaxLength(1)([{value:'a'},{value:'b'}])).toEqual( {maxLength: true});
-    expect(await Validators.MaxLength(3)(undefined)).toEqual(null);
+  it('should validate "maxLength" Input correctly', async () => {
     expect(await Validators.MaxLength(3)('abcd')).toEqual({ maxLength: true });
+    expect(await Validators.MaxLength(3)(stringWithValue)).toEqual(null);
+    expect(await Validators.MaxLength(3)(null)).toEqual(null);
+    expect(await Validators.MaxLength(3)(undefined)).toEqual(null);
+    expect(await Validators.MaxLength(3)(emptyString)).toEqual(null);
+    expect(await Validators.MaxLength(3)(objectWithValue)).toEqual(null);
+    expect(await Validators.MaxLength(3)(emptyObject)).toEqual(null);
+    expect(await Validators.MaxLength(3)(emptyArray)).toEqual(null);
+    expect(await Validators.MaxLength(2)(arrayWithValues)).toEqual(null);
+    expect(await Validators.MaxLength(1)(arrayWithValues)).toEqual( {maxLength: true});
```

are minlength 0 and maxlength 0 something that should exist? or should we throw an exception?

---

Added an exception